pages/about.html

```diff
@@ -5,8 +5,38 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&family=Montserrat:ital,wght@0,400;1,700&display=swap" rel="stylesheet">
+  <link href="//db.onlinewebfonts.com/c/0fddb08144cea533dcd3d39388b7ffbe?family=GrandSlang+Roman" rel="stylesheet" type="text/css"/>
 </head>
 <body>
+
+  <header>
+    <nav class="block">
+        <ul class="home-nav">
+            <li><a href="home.html">Home</a></li>
+            <li><a href="about.html">About</a></li>    
+            <li><a href="projects.html">Pojects</a></li>    
+            <li><a href="contact.html">Contact</a></li>   
+        </ul>
+    </nav>
+  </header>
+
+  <main class="about-parent-container">
+      <article class="about-section">
+        <h1> Ana ... who?</h1>
+        <p> 
+          write about myself here
+        </p>
+      </article>
```

Comment:
This _probably_ shouldn't be an `<article>` since it likely doesn't stand on its own.  I guess it depends on how _much_ you write about yourself there later.